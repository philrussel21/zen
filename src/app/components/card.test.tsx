import {create} from 'react-test-renderer';
import Card from './card';

test('snapshot', () => {
  const render = create(<Card title="Test" description="Test" />).toJSON();
  expect(render).toMatchSnapshot();
});
