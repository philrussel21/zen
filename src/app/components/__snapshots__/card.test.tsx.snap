// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<article
  className="p-8 rounded-2xl bg-white"
>
  <h3>
    Test
  </h3>
  <p>
    Test
  </p>
</article>
`;
